#! /bin/sh -e
## 001_dag2html.1.dpatch by  <bubulle@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Tim Dijkstra patch for proper handling of autocomments

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@

diff -Nru geneweb-4.10.orig/dag2html/dag2html.1 geneweb-4.10/dag2html/dag2html.1
--- geneweb-4.10.orig/dag2html/dag2html.1	2004-12-14 10:53:09.000000000 +0100
+++ geneweb-4.10/dag2html/dag2html.1	1970-01-01 01:00:00.000000000 +0100
@@ -1,78 +0,0 @@
-.\" This program is free software; you can redistribute it and/or modify
-.\" it under the terms of the GNU General Public License as published by
-.\" the Free Software Foundation; either version 2 of the License, or
-.\" (at your option) any later version.
-.\"
-.\" This program is distributed in the hope that it will be useful,
-.\" but WITHOUT ANY WARRANTY; without even the implied warranty of
-.\" MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-.\" GNU General Public License for more details.
-.\"
-.\" You should have received a copy of the GNU General Public License
-.\" along with this program; if not, write to the Free Software
-.\" Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-.\"
-
-.TH DAG2HTML 1 "February 8, 2004" "dag2html" "Generates HTML code from genealogic DAGs"
-
-.SH NAME
-dag2html \- generates HTML code from genealogic DAGs
-
-.SH SYNOPSIS
-.B dag2html
-[\fIOPTION\fR]
-\fIFILE\fR
-
-.SH DESCRIPTION
-.B Dag2html
-generates HTML code from genealogic DAGs
-
-Generates HTML code from a DAG. It is well adapted for genealogic DAGs
-(it is used in the software GeneWeb, same author) but may be used for
-any kind of DAG.
-
-.SH OPTIONS
-
-.TP
-.BR -b\ <int>
-table border
-
-.TP
-.BR -c
-char output
-
-.TP
-.BR -i
-apply algorithm from bottom to top
-
-.TP
-.BR -n
-no optimization
-
-.TP
-.BR -g
-no group
-
-.TP
-.BR -v
-print version number and exit
-
-.TP
-.BR -help,\ --help
-display this list of options
-
-.TP
-.BR -h , \ --help
-should tell you about what the command line options do, but doesn't yet :)
-
-.SH AUTHOR
-Written by Daniel de Rauglaudre <daniel.de_rauglaudre@inria.fr>.
-This manual page was originally written by Cyril Bouthors <cyril@bouthors.org>.
-
-.SH COPYRIGHT
-Copyright (c) 1998-2005 INRIA
-.br
-This is Free Software; this software is licensed under the GPL version
-2, as published by the Free Software Foundation.  There is NO
-warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR
-PURPOSE.
